jekyll_files/Gemfile

```diff
@@ -0,0 +1,8 @@
+source "https://rubygems.org"
+gemspec
+
+# gem "jekyll-github-metadata", ">= 2.15"
+
```

Comment:
were all of these files autogenerated/part of the template?
- have you checked that each one is necessary (and should be in source control)?

Reply:
The Gemfile is necessary*. The Gemfile.lock is not necessary*. The Gemspec is necessary*. Each directory contains files which are necessary*. Some directories contain files which are not necessary*.

* "necessary" means that `bundle exec jekyll serve` will fail without the file. Maybe the file can be extricated from the dependency tree by judicious removal of import statements.  "Not necessary" means that the command will serve a fully functional documentation page, but the files might be useful in the future (e.g., a dark stylesheet)